data.json

```diff
@@ -2090,6 +2090,20 @@
             "furl": "http://www.mikeash.com/pyblog/rss.py",
             "hnp": 2529
         },
+        {
+            "name": "Mike Crittenden",
+            "url": "https://critter.blog/",
+            "tags": [
+                "software",
+                "management",
+                "productivity",
+                "happiness",
+                "leadership",
+                "teams",
+                "work"
+            ],
+            "furl": "https://critter.blog/feed/",
```

Comment:
Thanks for your contribution, could you remove "," from at the end.

Reply:
Done, thank you!